parser_library/src/parsing/grammar/ca_expr_rules.g4

```diff
@@ -186,48 +186,59 @@ expr_comma_c returns [std::vector<ca_expr_ptr> ca_exprs]
 		$ca_exprs = std::move($tmp.ca_exprs);
 	};
 
-
-
-created_set_body returns [concat_point_ptr point]
-	: ORDSYMBOL
-	{
-		collector.add_hl_symbol(token_info(provider.get_range( $ORDSYMBOL),hl_scopes::var_symbol));
-		$point = std::make_unique<char_str_conc>($ORDSYMBOL->getText(), provider.get_range($ORDSYMBOL));
-	}
-	| IDENTIFIER
-	{
-		collector.add_hl_symbol(token_info(provider.get_range( $IDENTIFIER),hl_scopes::var_symbol));
-		$point = std::make_unique<char_str_conc>($IDENTIFIER->getText(), provider.get_range($IDENTIFIER));
-	}
-	| NUM
-	{
-		collector.add_hl_symbol(token_info(provider.get_range( $NUM),hl_scopes::var_symbol));
-		$point = std::make_unique<char_str_conc>($NUM->getText(), provider.get_range($NUM));
-	}
-	| var_symbol											{$point = std::make_unique<var_sym_conc>(std::move($var_symbol.vs));}
-	| dot													{$point = std::make_unique<dot_conc>();};
-
-created_set_body_c returns [concat_chain concat_list]
-	: (cl=created_set_body {$concat_list.push_back(std::move($cl.point));})+;
+created_set_body returns [concat_chain concat_list]
```

Comment:
Consider appending `_c` to maintain consistency with other methods returning `chain`s

Reply:
Alright